git_push.sh

```diff
@@ -19,7 +19,7 @@ if [ "$git_user_id" = "" ]; then
 fi
 
 if [ "$git_repo_id" = "" ]; then
-    git_repo_id="stitch-connect-python"
+    git_repo_id="stitch_connect_client"
```

Comment:
why this change?

Reply:
oh huh, the command on the readme is 
```
npx openapi-generator generate \
    -i https://raw.githubusercontent.com/NarrativeScience/stitch-connect-openapi/master/openapi.yml \
    -g python \
    -o . \
    --library asyncio \
    --package-name stitch_connect_client \
    --git-user-id NarrativeScience \
    --git-repo-id stitch_connect_client \
    -p packageVersion=0.1.1 \
    -t templates
```
but seems like it should be `stitch-connect-python`